examples/todo/imports/ui/profile/Profile.js

```diff
@@ -1,9 +1,94 @@
-import React from 'react';
+import React, { Fragment } from 'react';
+import { showAlert } from '@codeftw/future-web-ui-alert';
+import { Button, TextField } from 'material-ui';
+import { Save } from 'material-ui-icons';
 import { updateAppTitle } from '../components/uis';
 
-export const Profile = ({ loggedUser }) => (
-  <div>
-    {updateAppTitle('Profile')}
-    <p>{loggedUser.email}</p>
-  </div>
-);
+export class Profile extends React.Component {
```

Comment:
We are avoiding to have classes, we prefer only stateless components, can you change it? use `withState` from `recompose`.

Reply:
Profile.js is a copy past of AddTask.js, We should change in Profile and AddTask? But, I think the AddTask is a class because we need to use the method componentWillReceiveProps for edit task, @denihs tell me it. Is this right?

Reply:
we have lifecycle methods with recompose too